helix-term/src/keymap.rs

```diff
@@ -128,6 +132,8 @@ impl From<KeyTrieNode> for Info {
             let desc = match trie {
                 KeyTrie::Leaf(cmd) => cmd.doc(),
                 KeyTrie::Node(n) => n.name(),
+                // TODO
+                KeyTrie::Sequence(_) => "",
```

Comment:
I'm not sure what to do here instead of `""`

Reply:
Maybe something like `Doc of first command and …` with deliberate ellipses ? So with the example given in the first comment, the text shown will be `Open new line below selection and …`. Joining docs of all commands will make it long, and trying to fit in a custom description through the toml file doesn't seem like a good idea, and leaving it entirely blank isn't very helpful either.

Reply:
A `&str` is expected, so how would I manage the ownership?